Cannot import anything

Hi, I cannot import anything from `ngx-avatars`, neither module nor constants.

VScode and IntelliJ cannot auto-complete nor found any available import.
And the server (`ng`) doesn't compile because of the missing module.

I'm actually using Angular 12.2.12

Step to reproduce:
```sh
ng new app
cd app
npm install ngx-avatars --save
# in app.module.ts:
#   - add `import { AvatarModule } from 'ngx-avatars';`
#   - add `@NgModule({ ... imports: [... AvatarModule], ...})`
ng serve
```

The error:
```
Error: src/app/app.module.ts:3:30 - error TS2307: Cannot find module 'ngx-avatars' or its corresponding type declarations.

import { AvatarModule } from 'ngx-avatars';
```


For more details of the versions:

The Package npm intalling:
```json
{
  "dependencies": {
    "ngx-avatars": "^1.1.1"
  }
}
```

And my Angular env:

```

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 12.2.12
Node: 16.13.0 (Unsupported)
Package Manager: npm 8.1.0
OS: linux x64

Angular: 12.2.12
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1202.12
@angular-devkit/build-angular   12.2.12
@angular-devkit/core            12.2.12
@angular-devkit/schematics      12.2.12
@schematics/angular             12.2.12
rxjs                            6.6.7
typescript                      4.3.5
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,7 @@
 #IDE configuration files
 .idea
 .vscode
+.history
 
 # Dependency directory
 node_modules

--- a/angular.json
+++ b/angular.json
@@ -203,5 +203,5 @@
       }
     }
   },
-  "defaultProject": "ngx-avatars-demo"
+  "defaultProject": "ngx-avatars"
 }

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "testing",
+  "name": "ngx-avatars",
   "version": "1.0.0",
   "homepage": "https://github.com/Heatmanofurioso/ngx-avatars",
   "description": "README.md",
@@ -17,7 +17,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build",
+    "build": "ng build --configuration production",
     "postbuild": "node scripts/copy-artifacts.js",
     "test": "ng test",
     "lint": "ng lint",

--- a/projects/ngx-avatars/tsconfig.lib.prod.json
+++ b/projects/ngx-avatars/tsconfig.lib.prod.json
@@ -1,6 +1,6 @@
 {
     "extends": "./tsconfig.lib.json",
     "angularCompilerOptions": {
-        "enableIvy": true
+        "compilationMode": "partial"
     }
 }

--- a/scripts/copy-artifacts.js
+++ b/scripts/copy-artifacts.js
@@ -4,7 +4,7 @@ const path = require("path");
 const artifacts = ["README.md", "LICENSE"];
 artifacts.forEach(artifact => {
   const fromPath = path.resolve(__dirname, "..", artifact);
-  const toPath = path.resolve(__dirname, "..", "dist\\ngx-avatars", artifact);
+  const toPath = path.resolve(__dirname, "..", "dist/ngx-avatars", artifact);
 
   fs.readFile(fromPath, "utf-8", (err, data) => {
     if (err) {

